services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_scheduler.py

```diff
@@ -256,3 +256,17 @@ async def _task_cleanup_service_docker_resources(
         )
     except TaskAlreadyRunningError as e:
         raise HTTPException(status.HTTP_409_CONFLICT, detail=f"{e}") from e
+
+
+@router.post(
+    "/services/{node_uuid}/disk/emergency:free",
```

Comment:
Q: is emergency a resource now?

Reply:
So the idea is that the `reserved`(the new name) is a sub-resource of disk which is also a resource. Then this one gets freed `"/services/{node_uuid}/disk/reserved:free"`